package.json

```diff
@@ -20,7 +20,7 @@
     "react": "*"
   },
   "dependencies": {
-    "d3": "^3.5.17",
+    "d3": "*",
```

Comment:
made a bug - https://github.com/knitjs/knit/issues/18

should be able to set

```
"peerDependencies": {
  "d3": ">=3"
}
```

in survival/package.json when that is closed